docs/using/recipe_code/oracledb.dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM jupyter/base-notebook
+
+## Install java, javac and alien
+USER root
+RUN apt-get update --yes && \
+    apt-get install --yes --no-install-recommends software-properties-common && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*  && \
+    apt-get update --yes && \
+    add-apt-repository universe && \
+    apt-get update --yes && \
+    apt-get install --yes --no-install-recommends alien default-jre default-jdk openjdk-11-jdk libaio1 && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+## Then install Oracle SQL Instant client, SQL+Plus, tools and JDBC.
+## Note: You may need to change the URL to a newer version.
+## See: https://www.oracle.com/es/database/technologies/instant-client/linux-x86-64-downloads.html
+RUN mkdir "/opt/oracle"
+WORKDIR "/opt/oracle"
+RUN wget --progress=dot:giga https://download.oracle.com/otn_software/linux/instantclient/2111000/oracle-instantclient-basiclite-21.11.0.0.0-1.el8.x86_64.rpm && \
```

Comment:
I also noted Oracle provides permanent links to the latest packages.
Let's use them instead!

https://www.oracle.com/es/database/technologies/instant-client/linux-x86-64-downloads.html

Reply:
Thanks for the comment @mathbunnyru  

It would be cool to be able to make this recipe work to get the latest version of the Oracle packages and get the rigth folder name dinamically.

Unfortunately, I am not that sure how to make this work as you suggest and it would take much more time I could dedicate to it.

Reply:
Ah, we'll need to set the env variable based on the version.
I think you're right and it's difficult (or maybe impossible to do).
So, let's just use arg variables with fixed version.

Reply:
Could you please add arg variables?